### PR TITLE
W-10673273 adding merge input types

### PR DESCRIPTION
**Pull request by IsaacEldridge**

### CODE REVIEW

File: modules/ROOT/pages/merge-types.adoc [hunk 1]
```diff
@@ -274,6 +274,30 @@ price: float
 
 You can merge them so that in the unified schema there is only one `Amount` type that returns data from the Europe Sales API or the North America Sales API, depending on the query. Although both types are similar, their datasets are not joined.
 
+== Merge Input Types 
+
+Input types are POST request objects you use to modify information in your unified schema when you run mutations.
```
Comment (kbostrom) [inline modules/ROOT/pages/merge-types.adoc:279]: Ah, this clarifies my usage questions in # 67....

**Review by kbostrom — DISMISSED** [review]
Quick edit

File: modules/ROOT/pages/merge-types.adoc [hunk 2]
```diff
@@ -274,6 +274,30 @@ price: float
...
+customerId: String!
+OrderDate: String!
+product: Product
+Status: OrderStatus
+totalAmount: float
+----
+ 
+You can merge the `OrderInput` API into the `PurchaseInput` API in the unified schema to consolidate these types.
+
+You do not need to, nor can you, enable collaboration to merge input types.
```
[the remark this answers is not in the document]
Comment (IsaacEldridge) [reply modules/ROOT/pages/merge-types.adoc:299]: yep!

File: modules/ROOT/pages/merge-types.adoc (hunk 1 above)
Comment (IsaacEldridge) [inline modules/ROOT/pages/merge-types.adoc:279]:
```suggestion
Input types are POST, PUT, and DELETE objects you use to modify information in your unified schema when you run mutations.
```

**Review by tfoxy — APPROVED** [review]

**Pull request merged**